content/plugins/descriptions/argo-cd.yaml

```diff
@@ -0,0 +1,94 @@
+heading: 'Backstage Argo CD Plugin'
+# Keep it short
+lead: 'See Argo CD status in Backstage'
+
+seo:
+  # Don't forget to end with "| Roadie"
+  title: 'Backstage Argo CD Plugin | Roadie'
+  description: |
+    The Backstage Argo CD plugin integrates with your Argo CD instance api to show kubernetes status
+    information inside Backstage where it can be associated with your project.
+
+logo:
+  # This must be a relative path. It should start without a slash or with ./
+  # The overall size of the file should be approximately 200 by 200.
+  # The image in the file should be approximately 100 by 100 pixels. It needs space around it.
+  # The image will be made greyscale by Gatsby Image Sharp.
+  fileSystemPath: './content/assets/logos/argo-cd/argo-cd-logo.png'
+  # The dimensions are required. They are the outer dimensions of the image file.
+  width: 350
+  height: 350
+
+coverImage:
+  # This must be a relative path. It should start without a slash or with ./
+  fileSystemPath: './content/assets/argo-cd-plugin.png'
+  alt: 'A preview of Argo CD overview widget including kubernetes pod status.'
+
+# Instructions for someone who wants to use this plugin.
+# languages used here must be listed in the .babelrc
+gettingStarted:
+  # What will this step accomplish?
+  - intro: Install the plugin into Backstage.
+    language: bash
+    code: yarn add @roadiehq/backstage-plugin-argo-cd
+  - intro: Add proxy config
+    language: yaml
+    code: |
+      // app-config.yaml
+      proxy:
+        '/argocd/api':
+          target: https://&lt;your-argocd-instance>/api/v1/
+          changeOrigin: true
+          // only if your argocd api has self-signed cert
+          secure: false
+          headers:
+            Cookie:
+              $env: ARGOCD_AUTH_TOKEN
+  - intro: Add plugin to the list of plugins
+    language: typescript
+    code: |
+      // packages/app/src/plugins.ts
+      export { plugin as ArgoCD } from '@roadiehq/backstage-plugin-argo-cd';
+  - intro: 'Add argoCD widget to your overview page'
+    language: typescript
+    code: |
+      // packages/app/src/components/catalog/EntityPage.tsx
+      import {
+        ArgoCDDetailsWidget
+        isPluginApplicableToEntity as isArgoCDAvailable,
+      } from '@roadiehq/backstage-plugin-argo-cd';
+
+      const OverviewContent = ({ entity }: { entity: Entity }) => (
+        &lt;Grid container spacing={3} alignItems="stretch">
+          ...
+          {isArgoCDAvailable(entity) && (
+            &lt;Grid item md={6}>
+              &lt;ArgoCDDetailsWidget entity={entity} />
+            &lt;/Grid>
+          )}
+          ...
+        &lt;/Grid>
+      );
+  - intro: Add annotation to the yaml config file of a component
+    language: yaml
+    code: |
+      metadata:
+        annotations:
+          argocd/app-name: &lt;your-app-name>
+  - intro: Get and provide `ARGOCD_AUTH_TOKEN` as env variable in following format
+    language: yaml
+    code: |
+      `argocd.token=&lt;token>`
```

Comment:
The backticks are not required here.

It's also slightly confusing that the name of the env var is in the intro, but the token value is in the code sample. Could you instead please provide both the env var name and value in the code snippet.

Reply:
Done